packages/big-design/src/components/TableNext/SelectAll/SelectAll.tsx

```diff
@@ -3,13 +3,11 @@ import React from 'react';
 import { Checkbox } from '../../Checkbox';
 import { Flex, FlexItem } from '../../Flex';
 import { Text } from '../../Typography';
-import { TableExpandable, TableItem, TablePaginationProps, TableSelectable } from '../types';
+import { TableItem, TablePaginationProps, TableSelectable } from '../types';
 
 import { useSelectAllState } from './useSelectAllState';
 
 export interface SelectAllProps<T> {
-  expandedRowSelector?: TableExpandable<T>['expandedRowSelector'];
-  isExpandable: boolean;
```

Comment:
I guess these weren't being used?

Reply:
yeah, they were used to take into account child rows..